package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "audiobookshelf",
-  "version": "2.29.0",
+  "version": "2.30.0",
```

Comment:
Don't update the version number of the server in the PR. This will be done as part of the release process.